src/twisted/python/_release.py

```diff
@@ -482,13 +532,19 @@ def _writeMisc(self, fileObj, header, tickets):
         if not tickets:
             return
 
-        fileObj.write(header + '\n' + '-' * len(header) + '\n')
+        line = header + '\n' + '-' * len(header) + '\n'
+        if not isinstance(line, bytes):
```

Comment:
`line` is a variable that you just synthesized; you should know its type.  Rather than `isinstance`-checking it after creating it (a scenario which I believe can only occur on python2 where implicit encoding is allowed), you should use something like [`_matchingString`](https://twistedmatrix.com/documents/16.6.0/api/twisted.python.compat.html#_matchingString) for the `header`, and explicit `b` prefixes.